pages/database-management/monitoring.mdx

```diff
@@ -7,50 +7,102 @@ import { Callout } from 'nextra/components'
 
 # Monitoring
 
-In production systems, monitoring of applications is crucial, and that includes
-databases as well. Memgraph allows monitoring the database via WebSocket server and a
-HTTP server. 
+Monitoring of applications and databases is essential to be able to track the
+load, resources, logs, transactions, latency etc. 
 
-## Monitoring via WebSocket server
+Memgraph currently supports:
+
+- Logs tracking via WebSocket server 
+- Monitoring via HTTP server (Enterprise Edition)
```

Comment:
Add links to sections (anchors)

Reply:
Does it make sense to say "Telemetry" or "Metrics" instead of monitoring? Because monitoring is a big term that can have a lot in it, e.g. logs are in it, but logs are not on HTTP.